### DISCUSSION TITLE
v1.0.0: First release of the benchmark

### DISCUSSION THREAD

**Discussion by stephane-caron** [qpsolvers/qpbenchmark#75] [Announcements]
This is the first release of the QP solvers benchmark, which ships a ``qpsolvers_benchmark`` command-line tool contributed by @ZAKIAkram.

The goal of this benchmark is to help users compare and select QP solvers. Its methodology is open to [discussions](https://github.com/stephane-caron/qpsolvers_benchmark/discussions). The benchmark ships standard and community [test sets](#test-sets), as well as a ``qpsolvers_benchmark`` command-line tool to run any test set directly on your machine. For instance:

```console
qpsolvers_benchmark maros_meszaros/maros_meszaros.py run
```

The outcome from running a test set is a standardized report evaluating all [metrics](#metrics) of the benchmark across all available QP solvers. This repository also distributes [results](#results) from running the benchmark on all test sets using the same computer.

New test sets are [welcome](CONTRIBUTING.md). The benchmark is designed so that each test-set directory is standalone, so that the ``qpsolvers_benchmark`` command can be run on test sets from other repositories. Feel free to create ones that better represent the kind of problems you are working on.

Thanks to @aescande, @ottapav and @ZAKIAkram for contributing to this release :+1: 

### Added

* Allow non-lowercase solver names in the command line (thanks to @ottapav)
* Command-line tool and standalone test sets (thanks to @ZAKIAkram)

### Changed

* Plot: trim solutions that don't fulfill tolerance requirements
* Rename ``hist`` command to ``plot``
* Update to qpsolvers v3.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/qpsolvers/qpsolvers_benchmark/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by bodono** [comment]
Awesome work @stephane-caron !!
